public/js/controllers/game.js

```diff
@@ -300,9 +304,22 @@ angular.module('mean.system')
       $scope.selectList = (word) => {
         $scope.email = word;
       };
+
       $scope.drawCard = () => {
         game.drawCard();
       };
+
+      $scope.allGameRecords = () => {
+        $http.post('/api/games/history').then((games) => {
```

Comment:
Http calls are not supposed to be made by controllers, they should be abstracted by services